cli/exchange/business.go

```diff
@@ -281,6 +281,20 @@ func BusinessNewPolicy() {
 		`        {`,
 		`          "name": "",`,
 		`          "value": null`,
+		`        }`,
+		`      ]`,
+		`    }`,
+		`  ],`,
+		`  "secretBinding": [ /* ` + msgPrinter.Sprintf("A list of secret bindings for the secret names defined in the services, listed by service.") + ` */`,
+		`    {           `,
+		`      "serviceOrgid": "",         /* ` + msgPrinter.Sprintf("The org of the service.") + ` */`, 
+		`      "serviceUrl": "",           /* ` + msgPrinter.Sprintf("The name of the service.") + ` */`,
+		`      "serviceVersionRange": "",  /* ` + msgPrinter.Sprintf("The service version range.") + ` */`,
+		`      "secrets": [                /* ` + msgPrinter.Sprintf("The secret bindings.") + ` */`,
+		`        {      `,
+		`          "<service-secret-name>": "<vault-secret-name>"  /* ` + msgPrinter.Sprintf("The valid formats for vault secret names are:") + ` */`,
```

Comment:
Please change vault-secret-name to secret-provider-secret-name. And change the Sprintf string too.

Reply:
done